webui/react/src/pages/ModelRegistry.tsx

```diff
@@ -422,17 +424,38 @@ const ModelRegistry: React.FC = () => {
   return (
     <Page docTitle="Model Registry" id="models">
       <Section title="Model Registry">
-        <ResponsiveTable
-          columns={columns}
-          dataSource={models}
-          loading={isLoading}
-          pagination={getFullPaginationConfig({
-            limit: settings.tableLimit,
-            offset: settings.tableOffset,
-          }, total)}
-          showSorterTooltip={false}
-          size="small"
-          onChange={handleTableChange} />
+        {models.length === 0 && !isLoading ?
+          <div style={{
+            marginInline: 'auto',
+            marginTop: 150,
+            maxWidth: 350,
+            textAlign: 'center',
+          }}>
+            <div style={{ padding: 16 }}>
+              <Icon name="model" size="mega" />
+            </div>
+            <h4 style={{ marginBottom: 8 }}>No Models Registered</h4>
+            <p style={{
+              color: 'var(--theme-colors-monochrome-9)',
+              fontSize: 'var(--theme-sizes-font-medium)',
+            }}>
+              Track important checkpoints and versions from your experiments.{' '}
```

Comment:
**suggestion:** a slightly better way of introducing a gap is using HTML `&nbsp;` entity. `{' '}` involves JS where as `&nbsp;` is pure HTML.